eng/scripts/validate-docs-package.ps1

```diff
@@ -4,53 +4,84 @@ Validates packages by simulating docs CI steps
 
 .PARAMETER Package
 An object describing a package configuration to validate in the format used by
-docs CI onboarding format.
-
+docker image, which is the wrapping of docs CI build commands.
 Required attributes: 
   - name: The name of the package
-
   Supported optional attributes:
   - registry: The registry to use for the package if it's not NPM
-
 Example: 
-
 ```
 @{
   name = "@azure/attestation@dev";
   folder = "./types";
   registry = "<url>";
   ...
 }
-```
+.PARAMETER DocValidationImageId
+The image name for package validation in format of '$containerRegistry/$imageName:$tag'. 
+e.g. azuresdkimages.azurecr.io/jsrefautocr:latest
 
-.PARAMETER WorkingDirectory
-Supplied to `npm install ... --prefix $WorkingDirectory`. The place where the 
-node_modules folder is created and the package is installed
 #>
 
 param(
   [object] $Package,
+  [string] $DocValidationImageId,
   [string] $WorkingDirectory
 )
 ."$PSScriptRoot\..\common\scripts\common.ps1"
 
 function GetResult($success, $package, $output) { 
   return @{ Success = $success; Package = $package; Output = $output }
 }
+function FallbackValidation() {
+  $additionalParameters = @()
+  $prefixDirectory = "$env:AGENT_TEMPDIRECTORY/npmPackage" 
```

Comment:
Why are we using this over the workingdirectory passed in? It is generally not a good idea to depend on these DevOPS specific variables as it will make it difficult to run and test these things locally.

Reply:
I am trying to use env:AGENT_TEMPDIRECTORY